server/database/models/applicant-model.js

```diff
@@ -0,0 +1,126 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const applicantSchema = new Schema({
+  fullName: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    lowercase: true,
+    unique: true,
+    required: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  location: {
+    type: String,
+  },
+  address: {
+    type: String,
+  },
+  cohorts: [String],
+  gender: {
+    type: String,
+  },
+  mobileNumber: {
+    type: Number,
+    min: 10,
+    max: 18,
+  },
+  age: {
+    type: Number,
+    min: 18,
+    max: 115,
```

Comment:
I think its better to be less than this number, mabyee  60 
```suggestion
    max: 60,
```